commands/audit/sca/common.go

```diff
@@ -21,6 +21,11 @@ import (
 	xrayUtils "github.com/jfrog/jfrog-client-go/xray/services/utils"
 )
 
+const (
+	// Visual Studio inner directory to exclude when copying the project to a temporary directory during the dependency tree build process.
+	ExcludeForCopyVS = ".vs"
```

Comment:
```suggestion
	DotVsRepoSuffix = ".vs"
```
The const name should indicate about what is the const, not what it is used for. Think that we might use it for other purposes